src/cli.ts

```diff
@@ -21,7 +21,16 @@ export function run(): Promise<void> {
       type: Boolean,
     });
     var cli = commandLineArgs(argsWithHelp);
-    var cliOptions = cli.parse();
+
+    try {
+      var cliOptions = cli.parse();
```

Comment:
The parser doesn't fail at parsing supplied options. It throws a exception when it is provided with a parameter that its not configured to parse 
See: https://www.npmjs.com/package/command-line-args#cliparseargv--object
E.g. polyserve --bla value1 would cause polyserve with this PR applied to print out the usage information. The printing of usage if a parameter can not be understood is normal behaviour for unix programs.and i think it would be nice to have.